src/main/resources/import.sql

```diff
@@ -22,7 +22,7 @@ INSERT INTO public.extension VALUES (17, true, 'AtlasMap Data Transformation edi
 INSERT INTO public.extension VALUES (18, true, 'Runtime Server Protocol UI', 'https://redhat.gallerycdn.vsassets.io/extensions/redhat/vscode-rsp-ui/0.24.9/1694458765563/Microsoft.VisualStudio.Services.Icons.Default', 'redhat.vscode-rsp-ui');
 INSERT INTO public.extension VALUES (19, true, 'Quarkus', 'https://redhat.gallerycdn.vsassets.io/extensions/redhat/vscode-quarkus/1.17.2023100604/1696580225942/Microsoft.VisualStudio.Services.Icons.Default', 'redhat.vscode-quarkus');
 INSERT INTO public.extension VALUES (20, true, 'Tekton Pipelines', 'https://redhat.gallerycdn.vsassets.io/extensions/redhat/vscode-tekton-pipelines/1.1.0/1669943505057/Microsoft.VisualStudio.Services.Icons.Default', 'redhat.vscode-tekton-pipelines');
-INSERT INTO public.extension VALUES (21, true, 'Didact', 'https://redhat.gallerycdn.vsassets.io/extensions/redhat/vscode-didact/0.4.1/1641908771304/Microsoft.VisualStudio.Services.Icons.Default', 'redhat.vscode-didact');
+INSERT INTO public.extension VALUES (21, true, 'Deprecated - Didact', 'https://redhat.gallerycdn.vsassets.io/extensions/redhat/vscode-didact/0.4.1/1641908771304/Microsoft.VisualStudio.Services.Icons.Default', 'redhat.vscode-didact');
```

Comment:
The displayName and icon are actually updated from the marketplace, so this will last until the next hour, then will be reverted

Reply:
you can set active=false though, but it won't show on the dashboard